python/docs/tools.md

```diff
@@ -36,12 +36,14 @@ Tools extend agent capabilities beyond text processing, enabling interaction wit
 
 Ready-to-use tools that provide immediate functionality for common agent tasks:
 
-| Tool | Description | Use Cases |
-|------|-------------|-----------|
-| `DuckDuckGoTool` | Search for data on DuckDuckGo | Web searches, fact-checking, retrieving current information |
-| `OpenMeteoTool` | Retrieve weather information for specific locations and dates | Weather forecasts, historical weather data, travel planning |
-| `WikipediaTool` | Search for data on Wikipedia | Research, educational inquiries, fact verification |
-| `MCPTool` | Discover and use tools exposed by arbitrary [MCP Server](https://modelcontextprotocol.io/examples) | Integration with external tool ecosystems |
+| Tool             | Description                                                                                        | Use Cases                                                   |
+|------------------|----------------------------------------------------------------------------------------------------|-------------------------------------------------------------|
+| `DuckDuckGoTool` | Search for data on DuckDuckGo                                                                      | Web searches, fact-checking, retrieving current information |
+| `OpenMeteoTool`  | Retrieve weather information for specific locations and dates                                      | Weather forecasts, historical weather data, travel planning |
+| `WikipediaTool`  | Search for data on Wikipedia                                                                       | Research, educational inquiries, fact verification          |
+| `MCPTool`        | Discover and use tools exposed by arbitrary [MCP Server](https://modelcontextprotocol.io/examples) | Integration with external tool ecosystems                   |
+| `PythonTool`     | Run arbitrary Python code in the remote environment.                                               |                                                             |
+| `SandboxTool`    | Run your own Python function in the remote environment.                                            |                                                             |
```

Comment:
@jenna-winkler I was unsure of what to put in "Use Cases" since the TS docs don't have that column

Reply:
Also @jenna-winkler the TS docs have the following line about the CustomTool/SandboxTool: https://github.com/i-am-bee/beeai-framework/blob/a28c9046efecaab9006839368c7114c488df9ea3/typescript/docs/tools.md?plain=1#L121
but there is no equivalent place in our docs to add that